features/_your_results/limit_tax_relief.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Limit Tax Relief
+  In order to get a clear picture of the impact on my take-home pay.
+  As an employee contributing greater than £40,000 per year to my pension
+  I want the results to reflect the fact that I will not get tax relief
+  for any contribution over £40,000 per year.
+
+  Background:
+    Given I am on the Your Details step
+
+    Scenario: Limit tax relief when above yearly figures
+      Given I fill in my details:
+        | age | gender | salary | salary_frequency | contribution |
+        | 25  | male   | 80000  | per Year         | Full         |
+      And I click the Next button
+      And I fill in my contributions:
+        | your_contribution | employer_contribution |
+        | 60                | 1                     |
+      When I move on to the results page
+      Then I should see on the results page:
```

Comment:
What are these numbers about? It will be super helpful to have clear headers just like you've previously added to enhance comprehension at first reading.